lib/phoenix/pubsub/pg2_adapter.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Phoenix.PubSub.PG2Adapter do
```

Comment:
Let's just call it `PG2`? As users will be starting it directly, I think we can skip the `Adapter` bit.